client/dashboard/sites/settings-caching/index.tsx

```diff
@@ -124,6 +116,28 @@ export default function CachingSettings( { siteSlug }: { siteSlug: string } ) {
 	};
 
 	const renderForm = () => {
+		const fields: Field< CachingFormData >[] = [
+			{
+				id: 'active',
+				label: __( 'Enable global edge caching for faster content delivery' ),
+				Edit: ( { field, onChange, data, hideLabelFromVision } ) => (
+					<CheckboxControl
```

Comment:
It's pitiful that once we need a field to be disabled, we need a custom control 🥹

Reply:
I weep also. But least it's on their radar https://github.com/Automattic/wp-calypso/pull/103661#discussion_r2107188498